kafka_consumer/main.go

```diff
@@ -370,6 +381,10 @@ func (c *Consumer) ConsumeClaim(session sarama.ConsumerGroupSession, claim saram
 ClaimMessages:
 	for message := range claim.Messages() {
 		log.Info("Message claimed", zap.Int32("partition", message.Partition), zap.ByteString("key", message.Key), zap.ByteString("value", message.Value))
+		if len(message.Key)+len(message.Value) > kafkaMaxMessageBytes {
+			log.Fatal("kafka max-messages-bytes exceeded", zap.Int("max-message-bytes", kafkaMaxMessageBytes),
+				zap.Int("recevied-bytes", len(message.Key)+len(message.Value)))
+		}
```

Comment:
Is it possible that the message size of only one row changed exceeds the `kafkaMaxMessageBytes`? (Although the ycsb workload doesn't contain a wide table) In this test `max-message-bytes` is set to 4096